Documentation/In-depth/SystemRequirements/Index.rst

```diff
@@ -219,6 +219,17 @@ Example Configuration:
        try_files $uri $uri/ /index.php$is_args$args;
    }
 
+    # TYPO3 Backend URLs
+    location /typo3/ {
+        absolute_redirect off;
```

Comment:
This line is not needed imho. Not sure why Randy added it for DDEV. Maybe @bnf can leave a comment here too...

Reply:
This line is needed if a https proxy is in front of the Nginx so it can safely be removed. Not sure if we should provide a hint here for a proxy configuration, this is something more advanced and has not to be documented here in the first place imho.